xbmc/android/activity/XBMCApp.cpp

```diff
@@ -101,6 +102,7 @@ CXBMCApp::CXBMCApp(ANativeActivity* nativeActivity)
   : CJNIApplicationMainActivity(nativeActivity)
   , CJNIBroadcastReceiver("org/xbmc/kodi/XBMCBroadcastReceiver")
 {
+  m_xbmcappinstance = this;
```

Comment:
@Montellese is that something that could be done after 15.0 release? This PR fixes several issues and a "hefty" re-factoring isn't something we want right now i guess